packages/node_modules/@webex/internal-plugin-mercury/test/unit/spec/mercury-events.js

```diff
@@ -8,10 +8,11 @@ import sinon from '@webex/test-helper-sinon';
 import MockWebex from '@webex/test-helper-mock-webex';
 import MockWebSocket from '@webex/test-helper-mock-web-socket';
 import uuid from 'uuid';
-import promiseTick from '../lib/promise-tick';
 import lolex from 'lolex';
 import {wrap} from 'lodash';
 
+import promiseTick from '../lib/promise-tick';
```

Comment:
Did linting complain about this placement?

Reply:
I guess so.

I just have vscode set to fix on save. Either that or the new `--fix` on the pre-commit formatted it.